rock/python.rb

```diff
@@ -171,6 +171,22 @@ def self.resolve_python(ws: Autoproj.workspace,
         end
     end
 
+    def self.rewrite_python_shims(python_executable, root_dir)
+        shim_path = File.join(root_dir, "install","bin")
+        if !File.exist?(shim_path)
+            FileUtils.mkdir_p shim_path
+            Autoproj.warn "Rock.rewrite_python_shims: creating "\
```

Comment:
Won't that hit at least one time for every workspace ?

You could avoid having that warning by issuing it only if `python_executable` was already set.

Reply:
If I see it correctly in Autoproj::Workspace.setup at the stage of calling Rock.activate_python (now assuming init.rb) install/bin should be present. 
Hence, the warning will only show up if working with a corrupted or test workspace or outside an autoproj workspace.

Reply:
Oh yes you're right. Misread it as if you were testing the whole file path, not just the directory. Sorry.

Reply:
Anyways, all's good from my p.o.v.